ui/src/app/workflows/components/workflows-list/workflows-list.tsx

```diff
@@ -225,6 +226,18 @@ export class WorkflowsList extends BasePage<RouteComponentProps<any>, State> {
         this.fetchWorkflows(namespace, selectedPhases, selectedLabels, pagination);
     }
 
+    private countsByCompleted() {
+        const counts = {complete: 0, incomplete: 0};
+        this.state.workflows.forEach(wf => {
+            if (wf.metadata.labels && wf.metadata.labels['workflows.argoproj.io/completed'] === 'true') {
```

Comment:
Extract `'workflows.argoproj.io/completed'` as a common constant like it is done in the Go code?

Reply:
done